wp-sw-cache/wp-sw-cache-db.php

```diff
@@ -40,6 +37,22 @@ public static function on_uninstall() {
     }
   }
 
+  public static function update() {
```

Comment:
For some this function never gets called.  WTF.

I assumed this because the version is never set in the db.